.github/workflows/main-push-build-test-and-publish-release.yaml

```diff
@@ -21,20 +21,26 @@ on:
       - main
 
 jobs:
-  main-push-build-test-and-publish-release:
+  main-create-timestamp-and-tag:
     runs-on: ubuntu-latest
+    outputs:
+      timestamp: ${{ steps.create_timestamp.outputs.timestamp }}
     steps:
       - name: "Checkout source code of the repository"
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
+      - name: "Create timestamp"
+        id: create_timestamp
+        run: printf "timestamp=%s\n" "$(date +%Y%m%d-%H%M%S)" >> "$GITHUB_OUTPUT"
```

Comment:
duplicated in `biweekly-create-timestamp`

Reply:
I'm afraid that GitHub Actions output i.e. `$GITHUB_OUTPUT` can't be passed upwards, from a separated action to a job. I checked it before and now double-checked it as well.